src/profile/main/tag/Google AdWords Remarketing - Prod_QA-UID26.js

```diff
@@ -58,19 +58,37 @@ try {
             // Start Mapping
             for (d in utag.loader.GV(u.map)) {
                 if (b[d] !== undefined && b[d] !== "") {
+                    var isFlightConfirmation, bookingWindow;
                     e = u.map[d].split(",");
                     for (f = 0; f < e.length; f++) {
                         prefix = /^ecomm\.|^hotel\.|^edu\.|^flight\.|^hrental\.|^job\.|^local\.|^listing\.|^travel\.|^dynx\./.exec(e[f]);
+                        prefix_custom = "custom.";
                         if (prefix !== null){
                             prefix = prefix[0].slice(0, -1);
                             u.data.params[prefix] = u.data.params[prefix] || {};
                             u.data.params[prefix][e[f].substr(prefix.length + 1)] = b[d];
-                        } else if (e[f].indexOf("custom.") == 0) {
-                            u.data.google_custom_params[e[f].substr(7)] = b[d];
+                        } else if (e[f].indexOf(prefix_custom) == 0) {
+                            if (e[f].substr(prefix_custom.length) === "isFlightConfirmation") {
+                                isFlightConfirmation = b[d];
+                            } else {
+                                u.data.google_custom_params[e[f].substr(prefix_custom.length)] = b[d];
+                                if (e[f].substr(prefix_custom.length) === "bookingWindow") {
+                                    bookingWindow = parseInt(b[d]);
+                                }
+                            }
                         } else {
                             u.data[e[f]] = b[d];
                         }
                     }
+                    if (isFlightConfirmation !== "" && isFlightConfirmation === true) {
```

Comment:
We should already have an 'isFlightConfirmation' value in 'b'. You should just be able to do 
```
if (b["isFlightConfirmation"]) {
```
here and not need it on lines 71-73 above. This allows you to get rid of the if/else statement in the custom variable processing, as well.